traffic_monitor_golang/traffic_monitor/datareq/bandwidth.go

```diff
@@ -0,0 +1,18 @@
+package datareq
```

Comment:
you are missing the license file in a lot of these files, can you please add it?